### PR TITLE
A2 julius

### PR DESCRIPTION
**Pull request by julwil**
Increased statement coverage of LogisticUtils class from 0 to 30%

### CODE REVIEW

File: test/src/edu/stanford/nlp/objectbank/LineIteratorTest.java [hunk 1]
```diff
@@ -4,32 +4,44 @@
...
+                Assert.assertEquals("Wrong line", out, ans);
+            }
+        } catch (Exception e) {
+            Assert.fail("Probably too few things in iterator: " + e);
+        }
     }
-  }
 
+    @Test
+    public void testEnoughElements() {
```
Comment (RinorSefa) [inline test/src/edu/stanford/nlp/objectbank/LineIteratorTest.java:40]: Don't we have here Eager Test ?

File: test/src/edu/stanford/nlp/util/StringUtilsTest.java [hunk 2]
```diff
@@ -10,100 +10,112 @@
 
 public class StringUtilsTest {
```
Comment (RinorSefa) [inline test/src/edu/stanford/nlp/util/StringUtilsTest.java]: This is hard to review, what changes do we have here expect code style?

**Review by jvdassen — APPROVED** [review]
It looks all clean and good!

File: test/src/edu/stanford/nlp/objectbank/LineIteratorTest.java (hunk 1 above)
Comment (julwil) [reply test/src/edu/stanford/nlp/objectbank/LineIteratorTest.java:40]: @RinorSefa  How do you mean? I split the old test case into two. One of them is 'testEnoughElements' and the other one is 'testWrongLine' Maybe it's easier to see the changes from all commits. You click on 'Files changed' tab and then select 'view changes from all commits' view 
image

**Review by RinorSefa — APPROVED** [review]
Clean test codes.

**Pull request merged**